editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5351,9 +5356,9 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	v_scroll->connect("value_changed", this, "_update_scroll");
 	v_scroll->hide();
 
-	zoom_hb = memnew(HBoxContainer);
-	viewport->add_child(zoom_hb);
-	zoom_hb->set_begin(Point2(5, 5));
+	viewport->add_child(controls_vb);
+	controls_vb->set_begin(Point2(5, 5));
+
 	// Bring the zoom percentage closer to the zoom buttons
 	zoom_hb->add_constant_override("separation", Math::round(-8 * EDSCALE));
```

Comment:
If I am not mistaking, when we can, we keep the configuration of the node just after its creation, it's easier to read.
But that's a cosmetic change. :)